code/game/machinery/floor_light.dm

```diff
@@ -17,7 +17,7 @@ var/global/list/floor_light_cache = list()
 
 	var/damaged
 	var/default_light_power = 0.75
-	var/default_light_range = 3
+	var/default_light_range = 5
```

Comment:
why 5 and not 4? Seems closer to the pre-merge value

Reply:
didnt check pre-merge version, was compairing on the example map when i was testing for other prs, i've lowered it tho